docker/deployed/testnet/api/config.json

```diff
@@ -31,17 +31,17 @@
     "Analytics": {
         "FetchExtraBlockInfo": true
     },
-    "Backup" : {
-		"Enabled": true, 
-		"Dir": "backups",               
-		"Frequency": 240,  
-		"EnableVacuum": true,      
-		"EnableCompression": true,
-        "Pruning" : {
+    "Backup": {
+        "Enabled": true,
+        "Dir": "backups",
+        "Frequency": 240,
+        "EnableVacuum": true,
+        "EnableCompression": true,
+        "Pruning": {
             "Enabled": true,
             "KeepFiles": 5
         }
-	},
+    },
     "Chains": [
```

Comment:
The rationale for selecting `NewBlockPollFreq` is still quite aggressive to have a similar UX for users as the previous subscription mode. We could hand tune these values a bit further to drive costs down even more, but probably we'll start making tradeoffs with _reactivity_.

For now, my idea is to select values to don't make tradeoffs about UX and see how much this can help with costs. After that, we can keep meditating it makes sense to be less reactive considering UX tradeoffs (if any).

The idea is that the values are the following:
- For fast chains that also don't have an expected block production since they're creating blocks one per transaction (e.g., Optimism), we use 5s.
- For other chains that have an expected average block time (e.g., Eth mainnet, Polygon Mumbai, etc.), we select that block time or slightly less. The value shouldn't be perfect since block time isn't guaranteed and dances around an expected value.